content/agent/docker/apm.md

```diff
@@ -31,6 +31,7 @@ For example, the following command allows the Agent to receive traces from anywh
 docker run -d -v /var/run/docker.sock:/var/run/docker.sock:ro \
               -v /proc/:/host/proc/:ro \
               -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+              -p 8126:8126/tcp \
```

Comment:
Correct. Probably we should update this section to `-p 127.0.0.1:8126:8126/tcp` saying:

"to receive traces from anywhere" (->) "to receive traces from your host only"

I think this should be enough for most of the users, while exposing the port anywhere is only for advanced users with specific use cases. Other than that, all good to me!